lib/docx/body-reader.js

```diff
@@ -525,24 +525,37 @@ function BodyReader(options) {
     }
 
     function readDrawingElement(element) {
-        var blips = element
+        var picture = element
```

Comment:
I'm lifting up the picture node reference so we don't have to look it up multiple times. If this is premature, let me know.